ui/src/app/pages/workspace/workspace-about.tsx

```diff
@@ -252,6 +252,17 @@ export const WorkspaceAbout = fp.flow(
         : '';
     }
 
+    get workspaceBucketUrl(): string {
+      return this.state.workspace
+        ? 'https://console.cloud.google.com/storage/browser/' +
+            this.state.workspace.googleBucketName +
+            ';project=' +
+            this.state.workspace.googleProject +
```

Comment:
This should be a `?` instead of `;`.

While semicolons can be used as either [path parameters](https://skorks.com/2010/05/what-every-developer-should-know-about-urls/#:~:text=Each%20of%20the%20path%20segments%20can%20contain%20parameters%20which%20are%20separated%20from%20the%20segment%20by%20a%20%3B%20(semi%2Dcolon)) or [query param separators](https://en.wikipedia.org/wiki/Query_string#Structure:~:text=semicolons%20%22%3B%22%20are%20not%20recommended%20by%20the%20W3C%20anymore%2C%20see%20below), this is not recommended. In this case, you are incorrectly passing the `authUser` parameter since `&`s must be preceded by the `?` separator. While this works, this could break in the future.

TIL

Reply:
ahh, I got that URL from gcp console. Good point!

Reply:
Interestingly, GCP seems to use semicolon path parameters often. So there may be future cases to use them in GCP links - something to watch out for!